sourcecode/apis/contentauthor/app/CollaboratorContext.php

```diff
@@ -8,6 +8,15 @@
 use Exception;
 use Illuminate\Database\Eloquent\Model;
 
+/**
+ * @property  string $system_id
+ * @property string $context_id
+ * @property string $type
+ * @property string $collaborator_id
+ * @property string $content_id
+ * @property Carbon $timestamp
```

Comment:
`timestamp` is not Carbon. Only `created_at` and `updated_at` will be cast by default